clients/java/dkv-client/src/main/java/org/dkv/client/DKVClient.java

```diff
@@ -109,6 +109,30 @@
      */
     void put(byte[] key, byte[] value, long expiryTS);
 
+    /**
+     * Associates the list of specified value with the specified key
+     * inside DKV database. Also sets the corresponding expiryTS of the key
+     * to the provided value. If this association already exists in the database,
+     * the old value is replaced with the given value.
+     *
+     * @param items List of KV.Strings to be persisted
+     * @throws DKVException if the underlying status in the response from
+     * the database is an error status
+     */
+    void put(KV.Strings ... items);
```

Comment:
why do we need strings apis ? just byte[] won't suffice ?

Reply:
We offer both string & byte api's in java client.